data_structures/linked_list.cpp

```diff
@@ -132,3 +136,7 @@ int main() {
 
     return 0;
 }
+catch(...)
+{
+	std::cerr<<"\nWrong Input!";
```

Comment:
Is it really needed to throw an error?
Why not simply re-run the `do` loop again?